IPython/lib/latextools.py

```diff
@@ -15,7 +14,12 @@
 from traitlets.config import get_config
 from traitlets.config.configurable import SingletonConfigurable
 from traitlets import List, Bool, Unicode
-from IPython.utils.py3compat import cast_unicode, cast_unicode_py2 as u
+from IPython.utils.py3compat import cast_unicode, cast_unicode_py2 as u, PY3
+
+if PY3:
```

Comment:
Also, re: "explicit is better than implicit", that's also why I think it should be an *explicit* check for the availability of `encodebytes`, rather than an *implicit* check based on the version flag.